corehq/apps/cleanup/management/commands/fix_forms_with_missing_xmlns.py

```diff
@@ -134,13 +134,16 @@ def fix_apps(unique_id_to_xmlns_map, app_to_unique_ids_map, log_file, dry_run):
             app = get_app(domain, app_id)
             for build in [app] + get_saved_apps(app):
                 for form_unique_id in form_unique_ids:
-                    set_xmlns_on_form(
-                        form_unique_id,
-                        unique_id_to_xmlns_map[form_unique_id],
-                        build,
-                        log_file,
-                        dry_run
-                    )
+                    if unique_id_to_xmlns_map.get(form_unique_id):
```

Comment:
ah right don't know why i did that. thank you. for this case it doesn't matter (and it's a one off), but will keep it in mind